Common.mk

```diff
@@ -717,7 +722,7 @@ help-list:
 add-generated-help-block: # Add or update generated help block to document project make file and support shell auto completion
 add-generated-help-block:
 	$(BUILD_LIB)/generate_help_body.sh $(MAKE_ROOT) "$(BINARY_TARGET_FILES)" "$(BINARY_PLATFORMS)" "${BINARY_TARGETS}" \
-		$(REPO) $(if $(PATCHES_DIR),true,false) "$(LOCAL_IMAGE_TARGETS)" "$(IMAGE_TARGETS)" "$(BUILD_TARGETS)" "$(RELEASE_TARGETS)" \
+		$(REPO) $(if $(PATCHES_DIR),true,false) "$(LOCAL_IMAGE_W_HELM_TARGETS)" "$(IMAGE_W_HELM_TARGETS)" "$(BUILD_TARGETS)" "$(RELEASE_TARGETS)" \
```

Comment:
Does this change end up making so the generated help is the same as it was?

Reply:
That was the goal and I confirmed that was the case by testing a few repos. 

Though by a closer look today, I believe the right thing to do is really to keep "$(LOCAL_IMAGE_TARGETS)" and "$(IMAGE_TARGETS)" here, and update all add-generated-help-block for projects since it doesn't make sense to have  helm targets in the image help block anymore. So I reverted the change I made here.

I can do a separate PR just to run through a bulk update for the add-generated-help-block if needed. Or if you think project owners can run them as they go, that work for me too.